_topics/assumeutxo.md

```diff
@@ -0,0 +1,46 @@
+---
+title: AssumeUTXO
+
+## Required.  At least one category to which this topic belongs.  See
+## schema for options
+categories:
+  - Consensus Enforcement
+  - P2P Network Protocol
+
+## Required.  Use Markdown formatting.  Only one paragraph.  No links allowed.
+excerpt: >
+  **AssumeUTXO** is a proposed mode for bootstrapping new full
+  nodes without having them verify old block chain history.
+
+## Optional.  Use Markdown formatting.  Multiple paragraphs.  Links allowed.
+extended_summary: |
+  Embedded in the code of the node would be a hash of the chain history
+  (UTXO set) up to a certain point.  The hash would be verified by the
```

Comment:
"chain history (UTXO set)" is misleading. The UTXO set is a snapshot of state at a certain time. It isn't the chain history.